src/handler.ts

```diff
@@ -93,9 +93,15 @@ export const handleRequest = async (
     context.component = body.component
     context.routePath = body.routePath || ''
 
+    let { settings } = body
```

Comment:
apologies for the misdirect internally, but making the `Manager.env` field public instead would be the desired approach here